release-notes/docs/3.12.0.md

```diff
@@ -0,0 +1,117 @@
+# 3.12.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [Supported software](#supported-software)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+<<< TODO >>>
+
+## Upgrade notes
+
+- This upgrade can be rolled out remotely. Users will download the new version in the background and be prompted to reload the app when it's ready. A small amount of data will be needed to download the new version.
+- Upgrading to this version does not require other applications to be upgraded. This release does not drop support for any hardware or software that works with the previous version.
+- This release modifies some CouchDB view definitions, so we recommend you Stage the upgrade and wait for the views to be rebuilt before rolling it out to minimize downtime.
+- It is recommended to test the upgrade on a clone of your server first to ensure it works well for your application.
+
+### Breaking changes
+
+None.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### Telemetry
+
+The CHT-Core is collecting new telemetry data:
```

Comment:
I think it's worth adding a couple of sentences explaining how you can benefit from this feature. This is the main feature for this release and was highly requested so it's worth explaining why it's awesome. Additionally it would be good to link to the documentation so people can investigate more.

Reply:
Done!